exchange/auction.go

```diff
@@ -125,6 +125,19 @@ func newAuction(seatBids map[openrtb_ext.BidderName]*pbsOrtbSeatBid, numImps int
 	}
 }
 
+// Calculate if the new bid (nbid) will win against the current winning bid (wbid) given preferDeals.
+func isNewWinningBid(nbid, wbid *openrtb.Bid, preferDeals bool) bool {
```

Comment:
Consider changing the variable names to be more obvious:
nbid -> bid (i think the 'new' part is implied)
wbid -> winningBid